core/src/main/java/org/elasticsearch/node/Node.java

```diff
@@ -739,6 +746,13 @@ BigArrays createBigArrays(Settings settings, CircuitBreakerService circuitBreake
     }
 
     /**
+     * Build the {@link SearchModule}. Overridden by tests.
+     */
+    protected SearchModule createSearchModule(List<SearchPlugin> plugins) {
```

Comment:
Hrm, I actually meant just `createSearchService()`? I don't think we need tests to override the entire SearchModule...